en/docs/apis/restapis/application.yaml

```diff
@@ -2658,6 +2658,10 @@ components:
           type: boolean
           example: false
           description: Decides whether user consent needs to be skipped during logout flows.
+        useExternalConsentPage:
+          type: boolean
+          example: false
+          description: Decides whether user needs to use external consent page.
```

Comment:
```suggestion
          description: Decides whether users should be presented with an external consent page.
```

Reply:
@sandushi can you confirm this suggestion?

Reply:
done